### DISCUSSION TITLE
Ntfy intergration

### DISCUSSION THREAD

**Discussion by tetra-archos** [monicahq/monica#7354] [Ideas]
Would anyone else find a [ntfy](https://ntfy.sh/) integration useful? it would be a good alternative to telegram for people who don't want  another messaging app just for one thing, ntfy also has some useful integrations that would be useful for automating with monica notifications.

**Comment by lenaxia** [comment]
I think apprise would be a much better implementation. https://github.com/caronc/apprise

1) it is free and self hosted
2) it provides integration with basically every major notification channel, including ntfy.

**Comment by terrible-python-coder** [comment]
I'm running version 4.1.2, and I can't figure out how to setup notifications! Yes I would love notifications. I personally use GOTIFY which uses webhooks. Are notifications to telegram supported on V4?
